docs/reference-implementation/Brownfield-Considerations.md

```diff
@@ -0,0 +1,43 @@
+# Brownfield scenario considerations
+
+While the initial usage of the Azure Mission-Critical reference implementation often times include deploying it into empty Azure subscription(s) aka greenfields, is it not limited to that. This guide describes some of the most common scenarios when deploying the Azure Mission-Critical reference implementation into existing environments aka brownfield scenarios.
+
+## Existing networking infrastructure
+
+The Azure Mission-Critical online reference implementation maintains its own virtual networks and subnets. It is supposed to be deployed fully independent of any existing infrastructure. For environments with existing networking infrastructure or requirements for connectivity to existing services either on-premises or in another virtual network, it is required to use the [Azure Mission-Critical - Connected](https://github.com/Azure/Mission-Critical-Connected/) reference implementation.
+
+Considerations when using existing networking infrastructure:
```

Comment:
question whether we need this part here at all? Or is it just enough to point people to -Connected?

Reply:
Yes, let's keep the intro and drop the considerations.

Reply:
Dropped.